### PR TITLE
Cherry-pick #10157 to 6.x: Add remaining memory metrics of pods in Kubernetes metricbeat module

### PR DESCRIPTION
**Pull request by jsoriano**
Cherry-pick of PR #10157 to 6.x branch. Original message: 

Some of the metrics available on the summary endpoint are missing in the kubernetes pod metricset. This PR attempts to add them back.

### CODE REVIEW

File: metricbeat/module/kubernetes/pod/_meta/fields.yml [hunk 1]
```diff
@@ -78,3 +78,35 @@
...
                 Memory usage as a percentage of the defined limit for the pod containers (or total node allocatable memory if unlimited)
+        - name: available
+          type: group
+          fields:
+            - name: bytes
+              type: long
+              format: bytes
+              description: >
+                Total memory available
+        - name: workingSet
```
Comment (ruflin) [inline metricbeat/module/kubernetes/pod/_meta/fields.yml]: working_set, page_faults? We normally don't use camelCase

Comment (jsoriano) [reply metricbeat/module/kubernetes/pod/_meta/fields.yml]: Umm, it was like this in the original PR, I'll open and backport a fix for this.

Comment (ruflin) [reply metricbeat/module/kubernetes/pod/_meta/fields.yml]: SGTM. Let's get this in and then also do a backport. Probably slipped through in the review on master :-)

Comment (jsoriano) [reply metricbeat/module/kubernetes/pod/_meta/fields.yml]: I have updated this branch with the fix for names so we backport both things at once :)

**Pull request merged**